data/fields/crossing/barrier.json

```diff
@@ -0,0 +1,16 @@
+{
+    "key": "crossing:barrier",
+    "type": "combo",
+    "label": "Barrier Arm",
+    "strings": {
+        "options": {
+            "full": "Full",
+            "double_half": "Double Half",
+            "half": "Half",
+            "no": "no",
+            "yes": "yes"
```

Comment:
These should be in Title Case like the other options.

Reply:
fixed in 4e60539d328c444bf37b94e96e189be99565b261, thanks for noticing!